hikari/events/shard_events.py

```diff
@@ -236,3 +237,22 @@ class MemberChunkEvent(ShardEvent):
     typing.Optional[builtins.str]
         The request nonce if specified, or `builtins.None` otherwise.
     """
+
+    @typing.overload
+    def __getitem__(self, index_or_slice: int, /) -> guilds.Member:
```

Comment:
Should this be called index instead?

Reply:
A slice isn't an index is it?

Reply:
No, but this overload takes an index.

Usually this would be a bad thing to do but the / operator prevents you referring to these by name, so it will remain compatible